src/stores/Comment/CommentStore.js

```diff
@@ -271,8 +271,7 @@ export const CommentStore = types
     }
 
     const listComments = flow((function* ({ mounted = { current: true } } = {}) {
-      self.setComments([]);
-
+      if (!mounted.current) self.setComments([]);
```

Comment:
I'm a bit iffy on reusing the mounted state like this, as its supposed to be a safeguard against setting data on unmounted components, that could otherwise be attempting to set data on a now removed part of the state tree. What was the use case of this change?

Reply:
The comments would blink off when making the list call after edit. I can maybe pass another optional value here